R/hiper_glm.R

```diff
@@ -1,11 +1,19 @@
 #' @export
-hiper_glm <- function(design, outcome, model = "linear"){
-  supported_model <- c("linear", "logit")
-  if(!(model %in% supported_model)){
-    stop(sprintf(""))
+hiper_glm <- function(design, outcome, model = "linear", option = list()) {
+  supported_model <- c("linear")
+  if (!(model %in% supported_model)) {
+    stop(sprintf("Model is not available"))
   }
   # TODO: find MLE.
-  hglm_out <- list()
+  if (model == "linear") {
+    if (is.null(option$mle_solver) == TRUE) {
```

Comment:
I'd give the option to specify the psuedo inverse explicitely as an option, e.g. add a 

`(is.null(option$mle_solver) == TRUE) | (option$mle_solver == "pseudo inverse")`

type thing